examples/text-generation/habana_model_adapter.py

```diff
@@ -0,0 +1,158 @@
+# coding=utf-8
```

Comment:
suggest to change the file name since it contains more then model adapter (parsing args also),
suggest something like lm_eval_utils